client/pages/about-us/main-section.js

```diff
@@ -1,9 +1,147 @@
 const MainSection = {
-  data: () => ({ message: "About Us" }),
-  template: `
-    <div class="main">
-      <h1>{{message}}</h1>
-    </div>
+template: `
+
+<div class="about-us-container">
+<h1>About us</h1>
+</div>
+<div class="about-us">
+    <div class = "about-hyf"> 
+        <p><strong>HackYourFuture Belgium </strong>is an initiative of Open Knowledge Belgium.
+    It is a coding school to support those who have limited access to education and labor market
+    Operated in Brussels and Antwerp. <br> <br>
+    <strong>HackYourFuture aims</strong> to enable 
+    refugees and disadvantaged people to build digital skills for a career in web development, 
+    facilitate the integration of newcomers, and address the shortage of qualified workforce in the IT sector.</p>
+    <br>
+    <a class="about-visit-website" href="https://hackyourfuture.be/">visit website</a>
```

Comment:
can you add ( target="_blank" ) after the class so when the user clicks on the button, will open HYF website in another tab/window